gef.py

```diff
@@ -648,7 +667,16 @@ def __getattr__(self, item):
     def __int__(self):
         return self.__addr
 
+    def tcachebin(self, i):
+        """Return head chunk in tcache[i]."""
+        heap_base = int(gdb.execute("p/x $_heap()", to_string=True).split()[-1], 0)
```

Comment:
I really don't like this kind of PEDA like code. Isn't `$_heap` accessible via `gdb.Parameter` instead? If not, let's merge as is, I'll add some more pythonic wrapper around that.

Reply:
See my TODOs: This is fixed with #366